server/controllers/home/index.js

```diff
@@ -0,0 +1,22 @@
+const { Location, Review, MainServices } = require('../../database/models');
+const { customError } = require('../errors');
+
+const getHomeData = async (request, response, next) => {
+  try {
+    const data = await Promise.all([{
+      location: await Location.findAll(),
+      services: await MainServices.findAll(),
+      topTenReviews: await Review.findAll({
```

Comment:
promise.all built to receive an array of promises, What you're doing here you give it one element as an object that will wait for promises to finish